dipy/tracking/local_tracking.py

```diff
@@ -143,7 +143,13 @@ def _generate_tractogram(self):
                         stream_status == StreamlineStatus.ENDPOINT or
                         stream_status == StreamlineStatus.OUTSIDEIMAGE):
                     continue
-                first_step = -first_step
+                if stepsF > 1:
+                    # Use the opposite of the first selected orientation for
+                    # the backward tracking segment
+                    first_step = F[0] - F[1]
+                    first_step = first_step / np.linalg.norm(first_step)
```

Comment:
let's try to avoid a zerodivision error via a small check

Reply:
thanks, fixed.